docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md

```diff
@@ -38,11 +38,11 @@ To view the resources associated with an Azure account, you must first add the a
 
 1. Log in to the Azure account whose resources you want to browse.
 
-1. Once logged in to an Azure account, the subscriptions associated with that account display. Select the check boxes for the account subscriptions you want to browse and then select **Apply**.
+1. Once logged in to an Azure account, the subscriptions associated with that account display. Choose the check boxes for the account subscriptions you want to browse and then select **Apply**.
```

Comment:
Microsoft style is to use "select" for a check box.

Reply:
Thank you! :) I mistook "check boxes" here for UI options. I'll fix it.